CGATPipelines/Pipeline/Control.py

```diff
@@ -68,29 +70,255 @@
 GLOBAL_OPTIONS, GLOBAL_ARGS = None, None
 
 
-def writeConfigFiles(pipeline_path, general_path):
+def getConfigPaths():
+    '''
+    Search the current and installation paths where the configuration files
+    live for the pipeline being called.
+    '''
+    # (Antonio) I've modified this section, see explanation and changes in the
+    # writeConfigFiles function above.
+    config_paths = []
+    # Get the name of the pipeline being called
+    # This could be:
+    # cgatflow readqc config
+    # pipeline_QTL config
+    # python /YYYY//XXXX/pipeline_XXXX.py config
+    try:
+        f = sys._getframe(1)
+        #caller = f.f_globals["__file__"] # cgatflow config
+        # globals will get Control.py
+        caller = f.f_locals["__file__"] # TO DO: cgatflow 
```

Comment:
@AndreasHeger 
Do we need globals here instead of locals?
The module's namespace will get Control.py and breaks the rest of the changes.
locals seems to work OK but needs more tests.
If globals is needed let me know if you can think of a better way of calling external code.

Reply:
Not sure. There might have been python2/python3 issues here, but can't remember. Now with python3 only, not a problem any more.